Projects/Embark/Sources/EmbarkPlans.swift

```diff
@@ -0,0 +1,185 @@
+import Apollo
+import Flow
+import Form
+import Foundation
+import hCore
+import hCoreUI
+import hGraphQL
+import Presentation
+import SnapKit
+import UIKit
+
+public struct EmbarkPlans {
+    @Inject var client: ApolloClient
+    let plansSignal = ReadWriteSignal<[GraphQL.ChoosePlanQuery.Data.EmbarkStory]>([])
+    @ReadWriteState var selectedIndex = 0
+    
+    var selectedPlan: ReadSignal<GraphQL.ChoosePlanQuery.Data.EmbarkStory?> {
+        $selectedIndex.withLatestFrom(plansSignal).map { (selected, plans) in
+            plans.enumerated().filter { (offset, plan) -> Bool in
+                return offset == selected
+            }.first?.element
+        }
+    }
+    
+    public init() {}
+}
+
+extension EmbarkPlans: Presentable {
+    public func materialize() -> (UIViewController, Disposable) {
+        let viewController = UIViewController()
+        let bag = DisposeBag()
+
+        let sectionStyle = SectionStyle(
```

Comment:
Should we add this as an extension to SectionStyle in hCoreUI maybe? Feels like this is used in quite a few places now.